translations/en.json

```diff
@@ -446,8 +446,8 @@
     "decline": "Decline",
     "decryption-failed-content": "An error occured decrypting your data. You might need to erase your old data and generate a new account. Tap “Apply” to erase or “Cancel” to try again",
     "default": "Default",
-    "default-account-placeholder": "Account {{number}}",
-    "default-watched-address-placeholder": "Watched address {{number}}",
+    "default-account-placeholder": "Account",
```

Comment:
this should be "Account Name" btw, feel free to check with @benjthayer so it will match Desktop 👍

Reply:
Oh yes, I missed the `Name` you previously mentioned. now it's fixed